install-deps.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/sh -e
```

Comment:
This breaks when sh == dash (which it is on ubuntu):

./install-deps.sh: 126: ./install-deps.sh: Syntax error: "(" unexpected